scripts/private-archive-preprocess/esa_cci_gravimetric_mass_balance_dtu/gmb_dtu_nc_to_gpkg.py

```diff
@@ -7,14 +7,15 @@
 
 DATASET_ARCHIVE_LOCATION = '/share/appdata/qgreenland-private-archive/esa_cci_gravimetric_mass_balance_dtu/'
```

Comment:
TODO: Extract and import private archive constant `Path('/share/appdata/qgreenland-private-archive')`

Reply:
Nevermind, I thought this was a task